apis/experimentapi/experimentapi.yaml

```diff
@@ -0,0 +1,661 @@
+                  Existing_Experiment:
+                    summary: If_Experiment_Already_Exists
+                    value: 
+                      id: ekstep.analytics.experiement.create
+                      ver: v1
+                      ts: '2020-11-23 06:42:39:001+0000'
+                      params:
+                        resmsgid: 8c244093-e19d-4694-a4ba-342bc0f8b721
+                        status: successful
+                      responseCode: OK
+                      result:
+                       expResponse:
+                         request: '#/components/schemas/Experiment-Request/example'
+                         err: Failed
+                         errorMsg: ExperimentId already exists.                    
+        
+          '400':
+            description: Bad Request for create experiment
+            content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/CreateExperiment-400BadRequest'
+                examples:
+                  EmptyRequest:
+                    summary: Request_Field_Missing
+                    description: Response example when request is missing
+                    value:
+                      $ref: '#/components/schemas/CreateExperiment-400BadRequest/example'
+                  RequiredRequestFieldsEmpty:
+                    summary: Required Fields Missing in Request
+                    description: Response example with all required fields
+                    value:
+                      id: ekstep.analytics.experiement.create
+                      ver: v1
+                      ts: '2020-11-23 06:42:39:001+0000'
+                      params:
+                        resmsgid: 8c244093-e19d-4694-a4ba-342bc0f8b721
+                        status: failed
+                        err: CLIENT_ERROR
+                        errorMsg:
+                          status: failed
+                          request.expId: Created By should not be empty
+                          request.name: Experiment Name should not be  empty
+                          request.createdBy: Created By should not be empty
+                      responseCode: CLIENT_ERROR
+                      result: null
+                  MissingDateFields:
+                    summary: Missing_Date_Fields 
+                    description: Response example with all required fields
+                    value:
+                      id: ekstep.analytics.experiement.create
+                      ver: v1
+                      ts: '2020-11-23 06:42:39:001+0000'
+                      params:
+                        resmsgid: 8c244093-e19d-4694-a4ba-342bc0f8b721
+                        status: failed
+                        err: CLIENT_ERROR
+                        errorMsg:
+                          status: failed
+                          data.startDate: Experiment Start_Date should not be empty
```

Comment:
Do we need underscores for the fields such as Start_Date. Also, do the field names be first letter capitalized?